configs/config.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional
+
+import yaml
+
+
+@dataclass
+class Config:
+    video_directory: Path
+    video_files: Optional[List[Path]]
+    data_directory: Path
+    data_name: Optional[str]
+
+    @classmethod
+    def load_config(cls):
+        return yaml.safe_load(Path("config.yaml").read_text())
+
+
+CONFIG = Config.load_config()
+
+if __name__ == "__main__":
+    print(CONFIG)
```

Comment:
is this intentional? or the start of a budding cli?

Reply:
oh this is a part of testing lol eventually moving it to a unit test.